modules/local/appendmetadata/main.nf

```diff
@@ -48,9 +48,20 @@ process APPEND_METADATA {
         def sample_key = clusters_rows[i][0]
         merged.add(clusters_rows_map[sample_key] + metadata_rows_map[sample_key][1..-1])
     }
+    // Remove empty columns
 
+    def transposed = merged.transpose()
```

Comment:
Could you comment that you're transposing so that the data is easier to work with?